x-pack/plugins/event_log/server/event_log_client_service.mock.ts

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { IEventLogClientService } from './types';
+
+const createEventLogStartMock = () => {
+  const mock: jest.Mocked<IEventLogClientService> = {
+    getClient: jest.fn(),
+  };
+  return mock;
+};
+
+export const eventLogStartMock = {
```

Comment:
```suggestion
export const eventLogServiceMock = {
```

Reply:
I've decided to rename it back to StartService to align with another comment you made.
It's what I had done originally but don't like the name `start` as it isn't very expressive. But client then limits this to being "about the client" which is limiting and will likely force a rename in the future.